cmd/server/command/start.go

```diff
@@ -249,37 +280,65 @@ func NewStart(startOptions *startOptions) *cobra.Command {
 						"commit-link", opts.Spec.Application.URL,
 						"commit-sha", opts.Spec.Application.SHA,
 						"releaser", opts.Releaser,
-						"type", "release")
-					logger.Infof("Release [%s]: %s (%s) by %s, author %s", opts.Environment, opts.Service, opts.Spec.ID, opts.Releaser, opts.Spec.Application.AuthorName)
+						"type", "release",
+					)
+					logger.Infof(
+						"Release [%s]: %s (%s) by %s, author %s",
+						opts.Environment,
+						opts.Service,
+						opts.Spec.ID,
+						opts.Releaser,
+						opts.Spec.Application.AuthorName,
+					)
 				},
-				"prometheus": func(ctx context.Context, opts flow.NotifyReleaseOptions) {
-					metricsObserver.ObserveRelease(metrics.Release{
-						Environment: opts.Environment,
+				"prometheus": func(_ context.Context, opts flow.NotifyReleaseOptions) {
+					metricsObserver.ObserveRelease(
+						metrics.Release{
+							Environment: opts.Environment,
+							Service:     opts.Service,
+							Releaser:    opts.Releaser,
+							Intent:      opts.Intent.Type,
+						},
+					)
+				},
+				"events": func(ctx context.Context, opts flow.NotifyReleaseOptions) {
+					logger := log.WithContext(ctx)
+					err = brokerImpl.Publish(ctx, &events.ReleasedEvent{
 						Service:     opts.Service,
-						Releaser:    opts.Releaser,
-						Intent:      opts.Intent.Type,
+						Namespace:   opts.Namespace,
+						ArtifactID:  opts.Spec.ID,
+						AuthorEmail: opts.Spec.Application.AuthorEmail,
+						AuthorName:  opts.Spec.Application.AuthorName,
+						Environment: opts.Environment,
+						Squad:       opts.Spec.Squad,
+						IntentType:  opts.Intent.Type,
 					})
+
+					if err != nil {
+						logger.Errorf("could not publish released_event. Error: %w", err)
+					} else {
+						logger.Debugf("published released_event for artifactId: %s", opts.Spec.ID)
+					}
```

Comment:
We could drop the `else` here as we tend to in Go 🙈 

```suggestion
						return
					}
					logger.Debugf("published released_event for artifactId: %s", opts.Spec.ID)
```